### ISSUE TITLE
Use rgl instead of gl-rs

### ISSUE THREAD

**Issue by splintersuidman** [splintah/game-of-life#19]
[`rgl`] is a safer and more idiomatic wrapper around OpenGL for Rust than `gl-rs`.

[`rgl`]: https://crates.io/crates/rgl

**Comment by syberant** [comment]
I started to work on this in the new rgl branch with 1bd2caf67cb269e97eec4d6a0275fff860d999f7, I found myself limited however by the fact that `rgl` is still very much WIP and can't do a lot of things. In order to go further or even switch completely we would have to fork the repository, make changes, and send a PR.

**DISCLAIMER:** I don't really know anything about OpenGL but the code seems to work.
**NOTE:** I had to change the version of `gl` to 0.6.0 to get it to stop panicking.

**Comment by splintersuidman** [comment]
Yes, I do think we have to wait a little while before rgl is mature enough to fit our needs.
